app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -7,4 +7,26 @@ def retire_now(requester = nil)
     update_attributes(:retirement_requester => requester)
     finish_retirement
   end
+
+  # Intend to be called by UI to display stdout. Therefore the error message directly returned
+  # instead of raising an exception.
+  def raw_stdout_via_worker(userid = User.current_user, format = 'txt')
```

Comment:
Maybe the argument order could be reversed here - the UI will never need to provide a different userid, but does need to provide a format.